kubernetes/samples/scripts/common/utility.sh

```diff
@@ -622,6 +761,47 @@ function createDomain {
   printSummary
 }
 
+#
+# Function to create a domain
+# $1 - boolean value indicating the location of the domain home
+#      true means domain home in image
+#      false means domain home on PV
+#
+function updateDomain {
+  if [ "$#" != 1 ]; then
+    fail "The function must be called with domainHomeInImage parameter."
+  fi
```

Comment:
Remove the if statement above as it is only called from domain-home-on-pv/create-domain.sh

Reply:
removed